app/views/users/index.html.erb

```diff
@@ -12,13 +12,13 @@
 
 <ul class="users">
   <div class="search_feature">
-    <form action="" id="form1">
+    <form action="" id="form1" onsubmit="return false">
```

Comment:
What does the `onsubmit="return false"` do?

Reply:
It stops the page from refreshing after hitting Enter.